pkg/controller/common.go

```diff
@@ -549,3 +569,12 @@ func max(a, b int) int {
 	}
 	return b
 }
+
+func contains(key string, items []string) bool {
```

Comment:
Consider `containsString` since Go doesn't allow overloading and `contains` is _very_ generic.